scopes/react/react/react.env.ts

```diff
@@ -131,18 +132,23 @@ export class ReactEnv implements Environment {
     });
   }
 
-  private getFileMap(components: Component[]) {
-    return components.reduce<{ [key: string]: string }>((index, component: Component) => {
+  private getFileMap(components: Component[], local = false) {
+    return components.reduce<{ [key: string]: ComponentMeta }>((index, component: Component) => {
       component.state.filesystem.files.forEach((file) => {
-        index[file.path] = component.id.toString();
+        index[file.path] = {
+          id: component.id.toString(),
+          homepage: local
+            ? `/${component.id.fullName}`
+            : `https:/bit.dev/${component.id.toString({ ignoreVersion: false }).replace('.', '/')}`,
```

Comment:
didnt you create a function to calculate the path from an id?

Reply:
yes, thanks!

Reply:
so much better!